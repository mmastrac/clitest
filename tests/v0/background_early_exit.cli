#!/usr/bin/env clitest --v0

background {
    $ printf "hi\n"
    ! hi
    *
}

$ sleep .1
*
