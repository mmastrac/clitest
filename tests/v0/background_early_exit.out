Running <root>/background_early_exit.cli ...
sleep .1
---
(no output)
---
✅ OK

(background) printf "hi\n"
---
hi
---
✅ OK

<root>/background_early_exit.cli PASSED
